.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤷💻🤦 Question/Forum
+    url: https://community.getgrist.com/
+    about: You can ask and answer other questions here
+  - name: 💬 Discord
+    url: https://discord.gg/xBjD9PU9
```

Comment:
This shows as invalid https://discord.gg/xBjD9PU9

Reply:
Thanks for having check that part, that should be fixed.

I took the link provided here, which seems correct this time: https://support.getgrist.com/newsletters/2023-11/#hang-out-with-us-on-discord